src/test/kotlin/com/exactpro/th2/lwdataprovider/TestSseEventSearchRequest.kt

```diff
@@ -0,0 +1,172 @@
+/*******************************************************************************
+ * Copyright 2021-2021 Exactpro (Exactpro Systems Limited)
```

Comment:
Please, correct the year in the new files
```suggestion
 * Copyright 2022 Exactpro (Exactpro Systems Limited)
```